loadtest.py

```diff
@@ -4,10 +4,10 @@
 import random
 import xml.etree.ElementTree as ET
 
-CLDR_SUBDIVISION_FILENAME = 'unicode_cldr_subdivision_codes.xml'
-TARGET_URL = os.environ['TARGET_URL']
-TEST_LOCATION_HEADER_NAME = os.environ['TEST_LOCATION_HEADER_NAME']
-TEST_USER_AGENTS = [
+_CLDR_SUBDIVISION_FILENAME = 'unicode_cldr_subdivision_codes.xml'
+_TARGET_URL = os.environ['TARGET_URL']
```

Comment:
I wonder if we should consider defaults for these?

e.g. `os.environ.get('TARGET_URL", "http://localhost:8000")`